source/game/common_data.h

```diff
@@ -475,19 +510,22 @@ struct CommonDataSub12 {
   int field_14;
 };
 
-struct CommonDataSub13 {
-  u32 field_C;
-  u32 field_10;
-  u32 field_14;
-  u16 field_18;
-  u16 field_1A;
-  u16 field_1C;
-  char field_1E;
-  char field_1F;
-  u32 field_20;
-  u32 field_24;
-  u32 field_28;
-};
+struct RespawnData {
+  Vec3 pos; 
+  s16 yaw;
+  s16 player_params;
+  s16 entrance_index;
+  u8 room_index;
+  s8 data;
+  u32 temp_swch_flags_maybe;
+  u8 field_1;
+  // Swaps to zero when an item can be used, 255 otherwise.
+  u8 btn_y_can_use_item;
+  u8 btn_x_can_use_item;
+  u8 btn_i_can_use_item;
+  //u32 stored_mask_id_maybe;
+  u32 temp_collect_flags_maybe;
+}; //0x20
```

Comment:
```suggestion
};
static_assert(sizeof(RespawnData) == 0x20);
```

(in modern C++ you can assert that the struct size is correct)